templates/quickstart-bitbucket-dc.template.yaml

```diff
@@ -1026,14 +1039,57 @@ Resources:
         Statement:
           - Effect: "Allow"
             Principal:
-              AWS: !GetAtt BitbucketClusterNodeRole.Arn
-            Action: "es:*"
+              AWS: "*"
+            Action: "es:ESHttp*"
             Resource: "*"
+      AdvancedSecurityOptions:
+        Enabled: true
+        InternalUserDatabaseEnabled: true
+        MasterUserOptions:
+          MasterUserName: bitbucket
+          MasterUserPassword: !Ref ElasticSearchPassword
+      DomainEndpointOptions:
+        EnforceHTTPS: true
+        # Should we enforce TLS 1.2?
```

Comment:
I guess based on the value below, the answer is `yes`? And we can remove the comment.

Reply:
I put that there because it's the most up to date value but since we're planning on doing TLS 1.2 across the quick starts it makes sense to use TLS 1.2 from the get-go

Reply:
I agree but don't we enforce the TLS1.2 with the policy below?

Reply:
Yep I put it there because I wanted to use the latest and the comment was kind of a question for review. I'll remove the comment now.